packages/instant/src/redux/analytics_middleware.ts

```diff
@@ -53,6 +53,30 @@ export const analyticsMiddleware: Middleware = store => next => middlewareAction
                 ).toString();
                 analytics.addUserProperties({ ethBalanceInUnitAmount });
             }
+            break;
+        case ActionTypes.UPDATE_SELECTED_ASSET:
```

Comment:
What if we default open to a certain asset?

Reply:
I see it should be handled by the `zero_ex_instant_provider.tsx` change above, correct?

Reply:
Yes, correct